earthfile2llb/converter.go

```diff
@@ -1361,6 +1361,11 @@ func (c *Converter) WithDockerRun(ctx context.Context, args []string, opt WithDo
 
 	enableParallel := allowParallel && c.opt.ParallelConversion && c.ftrs.ParallelLoad
 
+	opt.extraRunOpts = nil
```

Comment:
Nit: Is this line really necessary?

`opt.extraRunOpts = nil`

Feels like it could create more surprises than it addresses if the logic changes somehow in the future.

Reply:
good question -- I thought it was shared in the interrupter and would be re-used for multiple `WITH DOCKER`, but it turns out it's created each time and can be safely removed.